opentelemetry-api/exporter/TrackAdvancedException/src/main/java/com/example/TrackAdvancedException.java

```diff
@@ -49,24 +55,16 @@ private static void trackAdvancedException() {
         }
     }
 
-    private static void initOpenTelemetry() {
-        LogRecordExporter logRecordExporter = new AzureMonitorExporterBuilder()
+    private static OpenTelemetry initOpenTelemetry() {
+        AutoConfiguredOpenTelemetrySdkBuilder sdkBuilder = AutoConfiguredOpenTelemetrySdk.builder()
+            .addResourceCustomizer((resource, configProperties) ->
+                resource.merge(Resource.getDefault().toBuilder()
+                    .put(ResourceAttributes.SERVICE_NAME, "my cloud role name")
+                    .put(ResourceAttributes.SERVICE_INSTANCE_ID, "my cloud instance id")
+                    .build()));
+        new AzureMonitorExporterBuilder()
```

Comment:
should we prevent user from calling `new AzureMonitorExporterBuilder().build()` multiple times even though we have freeze to guard it? build will init all the exporters. is there a use case that calling it multiple times is necessary? Let's chat tomorrow. AzureMonitorExporter.builder()? An actual builder pattern?

Reply:
let me know if this is addressed now by the rename from `void build(...)` to `void install(...)`